group_vars/all/main.yml

```diff
@@ -2,6 +2,14 @@ composer_keep_updated: true
 composer_global_packages:
  - { name: hirak/prestissimo }
 apt_cache_valid_time: 3600
+apt_package_state: present
+apt_security_package_state: latest
+apt_dev_package_state: latest
+package_vars_wrong_format_msg: |
+  The following variables must be formatted as dicts:
+    {{ package_vars_wrong_format | to_nice_yaml | indent(2) }}
+
+  See: https://github.com/roots/trellis/pull/881
```

Comment:
I'd prefer that this `package_vars_wrong_format_msg` var were defined in `roles/common/defaults/main.yml`, where users won't have to think about it unless they are really digging in. I think of this var as more of a helper/utility than a variable users are likely to want to modify. This change would keep `group_vars/all/main.yml` more streamlined.

An alternative that may be more consistent with other validation tasks could be to put the message in a template lookup ([example](https://github.com/roots/trellis/blob/4a2a70af503674f4c8d5a42edf4093d14d89f7b0/roles/common/tasks/main.yml#L4)).

Reply:
Moved into `roles/common/templates/package_vars_wrong_format_msg.j2`